kubesae/pod.py

```diff
@@ -44,8 +44,45 @@ def get_current_database(c):
     pass
 
 
+@invoke.task()
+def get_db_name(c, hide=False):
+    command = (
+        f"kubectl --namespace {c.config.namespace} exec -i "
+        f"deploy/{c.config.container_name} -- printenv DATABASE_URL"
+    )
+    return c.run(command, hide=hide)
+
+
+@invoke.task()
+def get_db_dump(c, filename="database.dump"):
```

Comment:
This all works great. May I suggest that instead of "database.dump" as a default, we do something like

> `get_db_dump(c, filename=None):
> ...
> if not filename:
>     filename = f"{c.config.namespace}_database.dump"
> ...
> `

Mainly because if I have a `db.dump` laying around, I like to better understand where it came from.

Reply:
Great idea!